api.planx.uk/modules/gis/routes.ts

```diff
@@ -1,12 +1,10 @@
 import { Router } from "express";
 import { locationSearch } from "./service/index.js";
-import { hasArticle4Schema } from "./service/article4Schema.js";
 import { classifiedRoadsSearch } from "./service/classifiedRoads.js";
 
 const router = Router();
 
 router.get("/gis/:localAuthority", locationSearch);
-router.get("/gis/:localAuthority/article4-schema", hasArticle4Schema);
```

Comment:
Would argue there's maybe a future, separate use-case of this endpoint for suggesting council-specific data field autocomplete values in the editor at some point (right now `property.constraints.planning` only suggests the schema-typed data fields, but there's no reason an editor modal couldn't concatenate granular team-specific values when applicable?)

All to say should we keep it around as a utility endpoint, or just re-introduce it in the future if we need?

Reply:
Sorry - meant to come back to this comment!

Yep it's a very fair question. I'd say as it's currently just returning a `true`/`false` we can remove for now - but I can certainly see a future use case for getting this list as you suggest.